src/sentry/integrations/slack/webhooks/event.py

```diff
@@ -109,19 +181,23 @@ def on_link_shared(self, request: Request, slack_request: SlackDMRequest) -> boo
 
         data = slack_request.data.get("event", {})
 
+        logger_params = {
+            "integration_id": slack_request.integration.id,
+            "team_id": slack_request.team_id,
+            "channel_id": slack_request.channel_id,
+            "user_id": slack_request.user_id,
+            "channel": slack_request.channel_id,
+            **data,
+        }
+
         # An unfurl may have multiple links to unfurl
         for item in data.get("links", []):
             try:
                 url = item["url"]
-                slack_shared_link = parse_link(url)
```

Comment:
deleted this b/c it created a log for every unfurl and we don't track this currently.